week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -14,7 +14,16 @@
   6. What value does the "a" parameter take in the first "f1" call?
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
+/*My answers
+1. b is not declared
+2. I commented out line  
+3. 2 6 4 9 6 13 8
+4. twice
+5 three times
+6.a in the first f1 call is 8.
```

Comment:
This might be a bit unclear from the question. What do you think would be the value of the inner parameter?